Lib/cu2qu/ufo.py

```diff
@@ -35,6 +36,8 @@
 
 DEFAULT_MAX_ERR = 0.0025
 
+logger = logging.getLogger('cu2qu')
```

Comment:
@jamesgk don't we wanna use different logger names for different sub-modules? So that clients can selectively tune-in, mute, filter, etc. specific sub-loggers if they need to (e.g. like you did for fonttools subsetter in fontmake)?

```python
logger = logging.getLogger(__name__)
```